src/Components/Results/ResultsError/ResultsError.tsx

```diff
@@ -7,7 +7,7 @@ import { useConfig } from '../../Config/configHook';
 
 const ResultsError = () => {
   const { uuid, whiteLabel } = useParams();
-  const { email } = useConfig('feedback_links');
+  const { email } = useConfig('feedback_links') as { email: string };
```

Comment:
I added a generic that does this?
```suggestion
  const { email } = useConfig<{ email: string }>('feedback_links');
```